app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -149,6 +154,18 @@ export default {
       this.$emit('toggle-canned-menu', !this.isPrivate && updatedValue);
     },
     value(newValue = '') {
+      if (this.hasImgTag && this.hasImgTag.length !== 0) {
```

Comment:
This change will fix showing the image in the advance editor. @nithindavid @pranavrajs @fayazara Is there any other good for this issue?